tests/dummy/app/pods/select/template.hbs

```diff
@@ -172,6 +172,7 @@
       <div class="demo">
         {{frost-select
           data=data
+          selectedValue=externalSelectedValue
```

Comment:
This was added just so that the demo exercises the same code path that bunsen will.